cmd/limactl/disk.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/docker/go-units"
 	"github.com/lima-vm/go-qcow2reader"
 	"github.com/lima-vm/lima/pkg/nativeimgutil"
-	"github.com/lima-vm/lima/pkg/qemu/imgutil"
+	"github.com/lima-vm/lima/pkg/qemuimgutil"
```

Comment:
Eventually this should follow:
- https://github.com/lima-vm/lima/issues/3518

Not an urgent topic though